tests/componenttests/client/stubs/WebAudioPlayerModuleStub.cpp

```diff
@@ -18,11 +18,37 @@
  */
 
 #include "WebAudioPlayerModuleStub.h"
+#include <IIpcServer.h>
+#include <IIpcServerFactory.h>
 #include <gtest/gtest.h>
 #include <memory>
 #include <string>
 #include <vector>
 
+namespace
+{
+firebolt::rialto::WebAudioPlayerStateEvent_WebAudioPlayerState
```

Comment:
Make a common header file with this conversion for both unittests and component tests, like we do in MediaPipeline. This should be used by WebAudioPlayerModuleServiceTestsFixture.cpp too.